torch/_torch_docs.py

```diff
@@ -993,60 +993,71 @@ def merge_dicts(*dicts):
            r"""
 asarray(obj, *, dtype=None, device=None, copy=None, requires_grad=False) -> Tensor
 
-Converts :attr:`obj` into a tensor, sharing data and preserving autograd history
-if possible.
+Converts :attr:`obj` to a tensor.
 
 :attr:`obj` can be one of:
 
 1. a tensor
 2. a NumPy array
 3. a DLPack capsule
-4. a Python object that implements the buffer protocol
-5. a Python sequence
+4. an object that implements Python's buffer protocol
+5. a scalar
+6. a sequence of scalars
```

Comment:
I think we still have some work to do here, @ysiraichi. Would you do me a favor and look at the following and we can file a follow-up PR if we need to make some changes?

For point 6 here, as mentioned elsewhere, does passing a range actually work?

Reply:
Yes, it does work with ranges also.